arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -733,8 +733,6 @@ public void tracePostExecution(MessageFrame frame, Operation.OperationResult ope
     if (isExceptional()) {
       this.currentTraceSection()
           .exceptionalContextFragment(ContextFragment.executionProvidesEmptyReturnData(this));
-      this.squashCurrentFrameOutputData();
-      this.squashParentFrameReturnData();
     }
```

Comment:
`squashParentFrameReturnData` replaces the value of `MemorySpan returnData` provided to the parent frame in `exceptionalContextFragment`, which is
```java
MemorySpan(currentFrame.contextnumber())
```
which has a nonzero context number, with
```java
MemorySpan.EMPTY
```
which has a context number of `0`.